apps/address-book/app/components/App/Entities.js

```diff
@@ -41,7 +41,7 @@ const Entities = ({ entities, onNewEntity, onRemoveEntity }) => {
         }
 
         renderEntry={([ name, entryAddress, entryType ]) => {
-          const typeRow = ENTITY_TYPES.filter(row => row.name === entryType)[0]
+          const typeRow = ENTITY_TYPES.filter(row => row.name === entryType || row.name === '_custom')[0]
```

Comment:
Some thoughts:

* Strange that we do a `filter` and then `[0]` instead of `find`

* This find/filter logic feels strange to me. Like, "Where row name is the type, or just grab the single `_custom` one." I would expect us to save a whole array of custom types, with proper names. Then we merge the saved entity types with the hard-coded ones for `Individual` and `Organization`, and continue to find by name.

* I think it would simplify the code to use an object for ENTITY_TYPES, rather than an array.
   ```js
   { Individual: { bg: ..., fg: ... },
     Organization: { bg: ..., fg: ... },
     'Custom Label 1': { bg: ..., fg: ... },
     'Custom Label 2': { bg: ..., fg: ... },
     ...
   }
   ```

  Now that we're persisting custom types, rather than just using a hard-coded list, I would expect this object to come from `appState`.